CMakeLists.txt

```diff
@@ -179,30 +180,59 @@ check_library_exists(rt shm_open "" LMMS_HAVE_LIBRT)
 
 LIST(APPEND CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}")
 
-FIND_PACKAGE(Qt5 5.9.0 COMPONENTS Core Gui Widgets Xml REQUIRED)
-FIND_PACKAGE(Qt5 COMPONENTS LinguistTools QUIET)
+IF(WANT_QT6)
+	MESSAGE("Building with Qt6 is experimental, not guaranteed to succeed")
+	SET(QT_VERSION_MAJOR 6)
+	SET(LMMS_QT_MIN_VERSION 6.0.0)
```

Comment:
It might be good to require 6.2 to deal better with removal of X11 extras, but it's not a strict requirement.